frontend/packages/console-shared/src/components/status/Status.tsx

```diff
@@ -98,7 +99,7 @@ export const Status: React.FC<StatusProps> = ({
       return <StatusIconAndText {...statusProps} icon={<NotStartedIcon />} />;
 
     default:
-      return <>{status || DASH}</>;
+      return <>{status && _.isString(status) ? status : DASH}</>;
```

Comment:
This changes the output when the value is the empty string, `''`. Is that intentional?

Reply:
Hmm, no. It also changes if status was a number or boolean value. Let me think about this line a little more.